Bottom: fd1558a8b57ab4e2a247d5925028795c07f87aff
Top:    24fd1273b7b242eda8cd18da3d8362eb33129ecd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 01:30:53 +0000

Refresh of remove-pseudocode-for-assembly

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index bfa1e61..04d1160 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -465,7 +465,6 @@ object3d_from_contours (const POLYAREA *contours,
 
           if (ct->is_round)
             {
-
               face3d_set_cylindrical (faces[i], COORD_TO_STEP_X (PCB, ct->cx), COORD_TO_STEP_Y (PCB, ct->cy), 0., /* A point on the axis of the cylinder */
                                                 0., 0., 1.,                                                       /* Direction of the cylindrical axis */
                                                 COORD_TO_MM (ct->radius));
